### ISSUE TITLE
Setting to not show Release Notes on startup

### ISSUE THREAD

**Issue by dferber90** [warpdotdev/Warp#1723]
### Describe the solution you'd like?

```text
Introduce a setting that lets me disable showing the Release Notes popup when Warp starts.
```

### Is your feature request related to a problem? Please describe.

```text
I love that Warp is constantly getting better. However the tiny friction the release notes modal shown on startup adds is pretty much enough to make me switch to a terminal which doesn't do that.
```

### How important is this feature to you?

5 (Can't work without it!)

**Comment by warpdotdev-dx** [comment]
Will update you when we have some!

**Comment by crash0verride11** [comment]
Agree, this would be great. Prevents me from really using Warp with Alfred (or any automation really) as every other time it launches I have to redo the automation after dismissing the release notes window.

**Comment by Acen** [comment]
Another window to close out of before I can send a command is breaking my focus and making it so I can get less work done.

Pleeeeaaase make this disableable.

I've switched back to terminal because I literally can't even.

**Comment by dannyneira** [comment]
Hey Folks! 

Please :+1:  and subscribe to this request ( not my comment ) as it helps us gauge interest. If we get a lot of 👍 's it will help us to prioritize the feature request! 

We will post any updates here if/when we have them. Thanks for your patience!
